src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,49 @@
 
 class Animal {
   // write your code here
+  static alive = [];
+
+  health = 100;
+
+  constructor(name) {
+    this.name = name;
+  }
 }
 
 class Herbivore extends Animal {
   // write your code here
+  constructor(name) {
+    super(name);
+
+    this.hidden = false;
+    Animal.alive.push(this);
```

Comment:
DRY.
Move it to the constructor of the animal class, so all other animals will inherit it

Reply:
The task: 
"Create a `Herbivore` class.
Herbivore has a method of `hide`, which changes the `hidden` property of the beast and helps to hide from carnivores."